android/sdl_android/src/main/java/com/smartdevicelink/util/SdlAppInfo.java

```diff
@@ -49,16 +59,18 @@
     //FIXME we shouldn't be duplicating constants, but this currently keeps us from needing a context instance.
     private static final String SDL_ROUTER_VERSION_METADATA = "sdl_router_version";
     private static final String SDL_CUSTOM_ROUTER_METADATA = "sdl_custom_router";
+    private static final String SDL_OEM_VEHICLE_TYPE_METADATA = "sdl_oem_vehicle_type";
 
 
     String packageName;
     ComponentName routerServiceComponentName;
     int routerServiceVersion = 4; //We use this as a default and assume if the number doesn't exist in meta data it is because the app hasn't updated.
     boolean isCustomRouterService = false;
+    List<VehicleType> vehicleMakesList = new ArrayList<>();
     long lastUpdateTime;
 
 
-    public SdlAppInfo(ResolveInfo resolveInfo, PackageInfo packageInfo) {
```

Comment:
@kostyaBoss On line 61 you have removed the 2 argument constructor.  You may need to bring it back for backward compatibility.

Reply:
I don't think we need to keep the backward compatibility in this way because in this case the all new VehicleType information would be lost, because of null-context issue

Reply:
@kostyaBoss `SdlAppInfoTests.java` doesn't build because it is using the 2 argument constructor.  Can you fix the build issues in `SdlAppInfoTests.java`?